It looks like it bypassed signing the RPM, but it needs to bypass signing the DEB too:

```
[INFO] --- jdeb:1.4:jdeb (default) @ elasticsearch ---
[INFO] Passphrase was not encrypted
[INFO] Creating debian package: /home/hinmanm/es/elasticsearch/distribution/deb/target/releases/elasticsearch-2.3.0-SNAPSHOT.deb
[WARNING] Signing requested, but no key supplied
[ERROR] Failed to create debian package /home/hinmanm/es/elasticsearch/distribution/deb/target/releases/elasticsearch-2.3.0-SNAPSHOT.deb
org.vafer.jdeb.PackagingException: Failed to create debian package /home/hinmanm/es/elasticsearch/distribution/deb/target/releases/elasticsearch-2.3.0-SNAPSHOT.deb
	at org.vafer.jdeb.DebMaker.makeDeb(DebMaker.java:296)
	at org.vafer.jdeb.maven.DebMojo.execute(DebMojo.java:552)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:862)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:286)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:197)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: java.lang.NullPointerException
	at org.vafer.jdeb.signing.PGPSigner.getSecretKey(PGPSigner.java:146)
	at org.vafer.jdeb.signing.PGPSigner.<init>(PGPSigner.java:54)
	at org.vafer.jdeb.DebMaker.makeDeb(DebMaker.java:280)
	... 23 more
```

--- a/distribution/deb/pom.xml
+++ b/distribution/deb/pom.xml
@@ -111,7 +111,7 @@
                             <goal>jdeb</goal>
                         </goals>
                         <configuration>
-                            <signPackage>true</signPackage>
+                            <signPackage>${deb.sign}</signPackage>
                             <keyring>${gpg.keyring}</keyring>
                             <key>${gpg.key}</key>
                             <passphrase>${gpg.passphrase}</passphrase>

--- a/distribution/pom.xml
+++ b/distribution/pom.xml
@@ -51,6 +51,7 @@
         <gpg.keypath>${project.parent.basedir}/src/test/resources/dummyGpg</gpg.keypath>
         <gpg.keyring>${gpg.keypath}/secring.gpg</gpg.keyring>
         <gpg.passphrase>dummy</gpg.passphrase>
+        <deb.sign>true</deb.sign>
     </properties>
 
     <!-- PUT TEST ONLY DEPS HERE. Let individual distributions figure out what they want -->
@@ -297,8 +298,7 @@
         </profile>
         <profile>
             <!-- Activate me to skip signing the rpm. Signing the rpm requires a few dependencies and might not work
-              properly on some distributions. Its known to work on Centos-6, and, ironically, Ubunutu 14.04.
-              This doesn't attempt to skip sigining the deb because we don't seem to have any problems with that. -->
+              properly on some distributions. Its known to work on Centos-6, and, ironically, Ubunutu 14.04. -->
             <id>skipSign</id>
             <activation>
                 <property>
@@ -306,8 +306,10 @@
                 </property>
             </activation>
             <properties>
-                <!-- Clearing this properly prevents the rpm plugin from trying to sign the rpm. -->
+                <!-- Clearing this property prevents the rpm plugin from trying to sign the rpm. -->
                 <gpg.key></gpg.key>
+                <!-- Clearing this property because once we've cleared the above one we can't sign the deb either. -->
+                <deb.sign>false</deb.sign>
             </properties>
         </profile>
     </profiles>

